frontend-react/src/shared/Citation/Citation.tsx

```diff
@@ -0,0 +1,23 @@
+export interface CitationProp {
+    title?: string;
+    quote?: string;
+    author?: string;
+    authorTitle?: string;
+}
+
+export const Citation = ({ citation }: { citation: CitationProp }) => {
+    return (
+        <div className="padding-top-9">
+            <p data-testid="title" className="font-sans-lg text-bold">
+                {citation.title}
+            </p>
+            <p data-testid="quote" className="usa-intro">
+                "{citation.quote}"
+            </p>
+            <h4 data-testid="author">{citation.author}</h4>
```

Comment:
Need to replace with utility classes to change text look or allow element override.